Champions/Caitlyn/E.cs

```diff
@@ -32,12 +31,12 @@ public class CaitlynEntrapment : GameScript
             var dashCoords = current + dash;
             ApiFunctionManager.DashToLocation(owner, dashCoords.X, dashCoords.Y, 1000, true, "Spell3b");
             spell.AddProjectile("CaitlynEntrapmentMissile", trueCoords.X, trueCoords.Y);
-            spell.spellAnimation("Spell3_fallback", owner);
         }
 
-        public void ApplyEffects(Champion owner, Unit target, Spell spell, Projectile projectile) {
+        public void ApplyEffects(Champion owner, Unit target, Spell spell, Projectile projectile)
+        {
             var ap = owner.GetStats().AbilityPower.Total * 0.8f;
-            var damage = 80 + spell.Level * 50 + ap;
+            var damage = 80 + (spell.Level - 1) * 50 + ap;
```

Comment:
This could be easier to read as
```C#
var damage = 80 + ((spell.Level - 1) * 50) + ap;
```
as the order of calculation is more clear.

Not a serious issue and I'm fine with this, but something to think about 😄